nameserver/entry.go

```diff
@@ -190,8 +194,7 @@ func (es *Entries) tombstone(ourname mesh.PeerName, f func(*Entry) bool) Entries
 
 	tombstoned := Entries{}
 	for i, e := range *es {
-		if f(&e) && e.Origin == ourname {
-			e.tombstone()
+		if f(&e) && e.Origin == ourname && e.tombstone() {
```

Comment:
That was the case already.

I did actually investigate what it would take to change the log messages so they are only emitted on the transition to tombstone. It's a non-trivial change, so not worth it.